data/categories.txt

```diff
@@ -6485,6 +6485,56 @@ sw:Malazi
 fa:پناهگاه
 mr:निवारा
 
+amenity-shelter-basic_hut|@category_hotel
+en:3Bivouac Hut
+ar:كوخ إقامة مؤقتة
+be:хаціна-бівак
+bg:хижа за бивак
+ca:cabana bivac
+cs:bivakovací chata
+da:Bivuakhytte
+de:4Biwakschachtel
+el:καταφύγιο bivouac
+es:cabaña de vivac
+es-MX:cabaña vivac
+et:bivouac-küünla
+eu:bivouac borda
+fa:کلبه بیواک
+fi:bivouac-mökki
+fr:cabane de bivouac
+he:בקתת ביבוק
```

Comment:
```suggestion
he:בקתת לינה|בקתה
```
meaning: sleeping hut | hut

As far as I can tell, the automated translation "בקתת ביבוק" is nonsense. Searching it on DuckDuckGo gives results for bamboo hut, with bamboo (במבוק) being one letter away from that word.